src/index-writer/index.js

```diff
@@ -52,8 +57,11 @@ export class IndexWriter {
     }
   }
 
+  /**
+   * @returns {Iterable<SchemaName>}
```

Comment:
I changed the contract for this, but I don't think it was ever used previously so I think that's fine.